### ISSUE TITLE
Sequence summary table NA's 

### ISSUE THREAD

**Issue by ChloeDG** [nanoporetech/megalodon#45]
Hi there, 

Thank you for this great tool!! 

We are calling for mod bases 5mC and 5hmC in CpG context and have had great success so far. However our sequence summary text file is just a table of NAs. Not sure what is going on there? Command:  
`megalodon fast5/ --guppy-params "-d /home/hh/rerio/basecall_models"  --guppy-config res_dna_r941_min_modbases_5mC_5hmC_CpG_v001.cfg --guppy-server-path /opt/ont/guppy/bin/guppy_basecall_server --outputs basecalls mods per_read_mods mod_basecalls mappings mod_mappings --write-mods-text --mappings-format bam --reference consensus.fasta --mod-motif mh CG 0 --output-directory megalodon`

No errors or anything. We get all of the output we are expecting but the sequence_summary.txt is a massive NA table like this: 
```
filename	read_id	run_id	batch_id	channel	mux	start_time	duration	num_events	passes_filtering	template_start	num_events_template	template_duration	sequence_length_template	mean_qscore_template	strand_score_template	median_template	mad_template	scaling_median_template	scaling_mad_template
NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
```
It goes on like that for ages.....

Ubuntu 16.04 
Megadolon installed with conda

Thanks in advance for your help!

**Comment by marcus1487** [comment]
This is an issue with the fast5 interface for single vs multi FAST5 formats. The issue has been addressed internally and will be included in the next release. Thank you for flagging this issue!

**Comment by marcus1487** [comment]
This is fixed in release 2.2.
